apps/src/music/views/MusicLabView.tsx

```diff
@@ -300,6 +300,7 @@ const MusicLabView: React.FunctionComponent<MusicLabViewProps> = ({
                 clearCode={clearCode}
                 allowPackSelection={allowPackSelection}
                 skipUrl={skipUrl}
+                blocklyWorkspace={blocklyWorkspace}
```

Comment:
Optional: instead of passing the whole workspace reference through, could just pass a callback that calls `hideChaff`?

Reply:
Updated!